pressbooks/pressbooks-multi-institution
# fix: update permissions for institutional managers

Allows institutional managers to create new books when registered users are not allowed to and prevents them from accessing pages menu and editing pages on the root site. Fix for #217  

To test:
1. Go to Network Options -> Book & User Registration Settings and uncheck the 'Allow registered users to create and clone new books' option
2. Log in as an institutional manager and click the 'Create Book' link in the top nav bar. Observe that you can now create new books
3. Attempt to visit the pages menu on the root site or edit one of the pages. Observe that you see a 'you do not have permission' message
4. Visit the organize page on any book assigned to your institution and edit a chapter. Observe that you can still edit content in books that belong to your institution.
